torch/_decomp/decompositions.py

```diff
@@ -1262,3 +1262,20 @@ def fast_pow(x, ord):
         self = self.abs()
 
     return fast_pow(fast_pow(self, p).sum(dim, keepdim=keepdim), 1.0 / p)
+
+
+@register_decomposition(torch.ops.aten.kl_div_backward)
+def kl_div_backward(
+    grad_output: Tensor,
+    self: Tensor,
+    target: Tensor,
+    reduction: int = Reduction.MEAN.value,
+    log_target: bool = False
+) -> Tensor:
+    if not log_target:
+        grad_input = torch.where(target > 0, -target * grad_output, 0)
+    else:
+        grad_input = -target.exp() * grad_output
```

Comment:
This will fail due to https://github.com/pytorch/pytorch/issues/78867. Note that `kl_div` does not expect shapes of `input` and `target` to match.